.gitignore

```diff
@@ -4,5 +4,7 @@ proprietary/
 *.zip
 lambda/
 out.json
+chrome-win/
```

Comment:
What is this folder? Is this where you installed chromium? If so, could you install outside of this repository?